.github/workflows/rotki_ci.yml

```diff
@@ -7,24 +7,27 @@ on:
       - develop
       - bugfixes
 
+env:
+  PYTHON_VERSION: 3.9.13
```

Comment:
Question. Should we use `3.9` instead of setting the minor version? They include safety and fixes, so I don't know if it is nice to have it fixed or not

Reply:
No we shouldn't, and the reason is simple. We want to run tests against the same version of python that we use in packaging.

And in packaging we have to pin to a specific patch version because for macOS we don't use the `@action/setup-python`. 

The reason is that the versions of python that this action provides don't support earlier versions of macOS, so it crashes if you run the app in any earlier version than the one used in the runner.

For macOS we download a python version from python.org that supports down to 10.9 with a script where you have to specify the exact version down to patch.